examples/src/main/scala/org/apache/spark/examples/sql/SparkSQLExample.scala

```diff
@@ -60,7 +60,8 @@ object SparkSQLExample {
 
   private def runBasicDataFrameExample(spark: SparkSession): Unit = {
     // $example on:create_df$
-    val df = spark.read.json("examples/src/main/resources/people.json")
+    val df = spark.read.json(
+      "file:///" + System.getProperty("user.dir") + "/examples/src/main/resources/people.json")
```

Comment:
Oh, my bad. Thanks for pointing out. Then, how about passing the path of people.json/people.text via CLI?